.github/workflows/comms-prepare.yaml

```
@@ -34,8 +34,19 @@ jobs:
         with:
           sdk: ${{ matrix.sdk }}
 
-      - name: dart version
-        run: dart --version
+      - name: Install mobile-tools
+        uses: actions/checkout@v3
+        with:
+          repository: leancodepl/mobile-tools
+          path: mobile-tools
+
+      - name: Add mobile-tools to PATH
+        run: |
+          echo $GITHUB_WORKSPACE/mobile-tools/bin >> $GITHUB_PATH
```

Usually I'd recommend quoting everything that uses string substitution.
```suggestion
          echo "$GITHUB_WORKSPACE/mobile-tools/bin" >> "$GITHUB_PATH"
```
Same in other files.